tests/sqlparser_postgres.rs

```diff
@@ -834,6 +834,11 @@ fn parse_comments() {
     }
 }
 
+#[test]
+fn parse_quoted_identifier() {
+    pg_and_generic().verified_stmt(r#"SELECT "quoted "" ident""#);
+}
```

Comment:
This is definitely how postgres handles things

```sql
alamb=# SELECT "quoted "" ident" from t1;
ERROR:  column "quoted " ident" does not exist
LINE 1: SELECT "quoted "" ident" from t1;
             
```

Reply:
However, msql treats this differently

```sql
Database changed
mysql> SELECT "quoted "" ident" from t1;
+----------------+
| quoted " ident |
+----------------+
| quoted " ident |
| quoted " ident |
| quoted " ident |
| quoted " ident |
| quoted " ident |
+----------------+
5 rows in set (0.00 sec)

mysql> SELECT "quoted  ident" from t1;
+---------------+
| quoted  ident |
+---------------+
| quoted  ident |
| quoted  ident |
| quoted  ident |
| quoted  ident |
| quoted  ident |
+---------------+
5 rows in set (0.00 sec)

```

Reply:
`"` is used for strings in mysql unless `ANSI_QUOTES` is enabled https://dev.mysql.com/doc/refman/8.0/en/identifiers.html . Neither double quoted strings or  `ANSI_QUOTES` seems implemented in this crate and I don't think it is up to this PR to fix that.